src/forms/fields/password-field/index.jsx

```diff
@@ -175,37 +190,6 @@ const PasswordField = forwardRef((props, ref) => {
           {errorMessage}
         </Form.Control.Feedback>
       )}
-      {errorMessage && showPasswordTooltip && (
```

Comment:
Just for my confirmation. Have we decided, it will be removed from the register form too?

Reply:
Yes